src/components/Articles/ArticleDetail.jsx

```diff
@@ -79,6 +80,7 @@ const ArticleDetail = ({ article }) => (
               </button>
             </div>
             <DeleteArticleComponentModel />
+            <Comments comments={comments} showReplies={showReplies} replyDisplayState={replyDisplayState} slug={article.slug} postReplyToComment={postReplyToComment} />
```

Comment:
@ezrogha 
Can you reduce on the length of this line

Reply:
Thanks @felixkiryowa,
Feedback implemented